css/index.css

```diff
@@ -112,6 +149,117 @@ p {
     justify-content: center;
 }
 
```

Comment:
One of the good things about CSS is it cascades.  All these properties you put in each box could have been set in the parent container class="boxes".  All you would need to set is the color of each specific box such as .box1 { background: teal; }.  This would have saved 7 lines of code per box.